test/runtests.jl

```diff
@@ -69,6 +69,44 @@ end
                    8.848886704358202, 2.8697881579099085, 11.15107375630744, 11.8392578374927])
 end
 
+@testset "rankdeficient GLM" begin
+    # an example of rank deficiency caused by linearly dependent columns
+    num_rows = 100_000
+    dfrm = DataFrame()
+    dfrm[:x1] = randn(MersenneTwister(123), num_rows)
+    dfrm[:x2] = randn(MersenneTwister(456), num_rows)
+    dfrm[:x3]= 2*dfrm[:x1] + 3*dfrm[:x2]
+    dfrm[:y] = Int.(randn(MersenneTwister(9999999), num_rows) .> 0)
+    f1 = @eval(@formula(y ~ 1+x1+x2+x3))
+    @test_throws PosDefException fit(
```

Comment:
Common convention is to keep the first argument on the first line and align other arguments to it.

Reply:
Done!